### ISSUE TITLE
Missing rules for the lang attribute

### ISSUE THREAD

**Issue by JAWS-test** [w3c/accname#56]
Unfortunately, there is nothing in the entire document about handling the lang attribute within labels. However, correct speech recognition is important for all text-to-speech users and to comply with WCAG 3.1.1 and 3.1.2. Therefore, this should be added.

In addition to many open questions, I think of the following first:
* For which labelling forms is it possible to specify a language at all?
* Is it possible to change the language within a label or description to which reference is made (label for, aria-labelledby, aria-describedby). 
* Which specification in the lang attribute has priority: The one in the label or the one in the element.
* Should at least the lang specification be considered for the entire document.

In my opinion, the lack of an exact rule is also the reason why the language markup in JAWS is very inadequate (https://github.com/FreedomScientific/VFO-standards-support/issues?q=is%3Aissue+is%3Aopen+language).

See also: https://github.com/w3c/aria-practices/issues/1162

**Comment by scottaohara** [comment]
Do the following markup cover the uses cases you're asking about?

```html
<!-- example 1 -->
<label for="i" lang="fr">Qu'est-ce que tu veux?</label>
<input lang="en" id="i">

<!-- example 2 -->
<div id="d" lang="fr">Qu'est-ce que tu veux?</div>
<input lang="en" aria-labelledby="d">
```

**Comment by ZoeBijl** [comment]
Tested the above examples with Safari 12.1.2 and macOS 10.14.6. When moving linearly the labels are read with the correct synthesizer (French). Moving focus to the input (with or without a value) reads them with the wrong synthesizer (English).

**Comment by JAWS-test** [comment]
@scottaohara 

> Do the following markup cover the uses cases you're asking about?

Yes, but only for a part of my 3rd question: "Which specification in the lang attribute has priority: The one in the label or the one in the element." (for the 3rd question there should be also test cases with figcaption, legend, caption, aria-desribedby, enclosing label, th (as label for table cells) and so on.

For questions 1, 2 and 4 there should of course also be use cases. I can create them, if desired. 

See also: https://github.com/FreedomScientific/VFO-standards-support/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+language and https://github.com/w3c/aria-practices/issues/1163

**Comment by dd8** [comment]
It's also possible for a name / description to have mixed languages:

```
<span id='id-en' lang='en'>English</span>
<span id='id-fr' lang='fr'>Français</span>
<span id='id-is' lang='is'>íslenska</span>

<div role='button' aria-labelledby='id-en id-fr id-is'></div>
```

This should probably be a conformance error.

**Comment by LauraOH** [comment]
We're still encountering the impact of this limitation with Jaws 2026 and NVDA 2025.
